apis/python/src/tiledbsoma/sparse_nd_array.py

```diff
@@ -179,8 +180,13 @@ def read_sparse_tensor(
         if format not in ("coo", "csr", "csc"):
             raise NotImplementedError("format not implemented")
 
+        # The full shape of the data is a template for the shape of the answer.
+        # For example if the data is 2D and coords is (None, [5,7,9]), then
+        # output shape[0] will be taken from the full shape, and output shape[1] will
+        # be 3.
         with self._tiledb_open("r") as A:
-            shape = A.shape
+            shape = list(A.shape)
```

Comment:
no reason to convert the tuple to a list - the API below where you use it all accept a tuple AFAIK